addons/website/static/src/services/chat_window_service_patch.js

```diff
@@ -0,0 +1,13 @@
+/* @odoo-module */
+
+import { chatWindowService, ChatWindowService } from "@mail/web/chat_window/chat_window_service";
+
+import { patch } from "@web/core/utils/patch";
+
+chatWindowService.dependencies.push("website");
```

Comment:
I understand why it's there technically but dependency-wise this doesn't make a lot of sense. Chat window should be able to work without the website service too, and this part should just come as extra when both are present.